src/lib/utilities/pre-loader/pre-loader.scss

```diff
@@ -150,7 +150,7 @@
 
 .dark-theme #td-pre-loader {
   /* dark-theme */
-  background-color: map-get($mat-dark-theme-background, background);
+  background-color: mat-color($td-slate-dark, 1300);
```

Comment:
Same as above. In addition, this is slate 1500 now, another reason to reference a single variable.

Reply:
Right, the issue here was in fact that .bgc was not using `map-get($mat-dark/light-theme-background, background);` So I'll revert these lines here and fix .bgc in the overrides file